web/src/main/java/no/nav/modiapersonoversikt/infrastructure/tilgangskontroll/Tilgangskontroll.kt

```diff
@@ -18,22 +21,27 @@ fun AbacResponse.toDecision(denyReason: AbacResponse.() -> String): Decision = w
 class Policies {
     companion object {
         private val modiaRoller = setOf("0000-ga-bd06_modiagenerelltilgang", "0000-ga-modia-oppfolging", "0000-ga-syfo-sensitiv")
+        private val nasjonalTilgangRoller = setOf("0000-ga-gosys_nasjonal", "0000-ga-gosys_utvidbar_til_nasjonal", "0000-ga-pensjon_nasjonal_u_logg")
+        private val regionalTilgangRoller = setOf("0000-ga-gosys_utvidbar_til_nasjonal", "0000-ga-gosys_utvidbar_til_regional", "0000-ga-pensjon_nasjonal_m_logg")
+        private val kode6Roller = setOf("0000-ga-strengt_fortrolig_adresse", "0000-ga-gosys_kode6")
+        private val kode7Roller = setOf("0000-ga-fortrolig_adresse", "0000-ga-gosys_kode7")
```

Comment:
trur me for sikkerhetsskyld bør legge inn gamle pen-rollen også på kode 6 og 7.

Reply:
Om jeg forstår alfa-koden så skal i teorioen bare `ga-fortrolig-adresse` etc være i bruk, men ja, vi tar ikke sjansen på det. :D  

La også merke til at egenansatt/skjerming ikke blir sjekket her. Begge deler skal være fikset i: https://github.com/navikt/modiapersonoversikt-api/pull/754